tests/components/notify/test_command_line.py

```diff
@@ -53,6 +53,7 @@ def test_command_line_output(self):
 
             self.hass.services.call('notify', 'test', {'message': message},
                                     blocking=True)
+            self.hass.block_till_done()
```

Comment:
Seems blocking was working fine, the SERVICE_CALL_LIMIT was too low for a different test. I've updated it to wait longer and assertTrue on the call so we know its not timing out.